mdp_plan_exec/scripts/mdp_travel_time_estimator.py

```diff
@@ -27,17 +27,14 @@ def __init__(self,top_map):
         rospy.loginfo("MDP travel times estimator initialised.")
 
     def travel_times_to_waypoint_cb(self,req):
-        epoch=req.epoch
-        if req.epoch==0:
-            epoch=rospy.Time.now()
-        if epoch != self.last_epoch:
-            self.last_epoch=epoch
-            self.top_map_mdp.get_fremen_stats(epoch.secs)
-            #update model with fremen
-            self.top_map_mdp.write_prism_model(self.directory+self.file_name)
+        print "ENTERD SERVICE"
+        if req.epoch != self.last_epoch:
```

Comment:
I'm not sure we'll often get this case, unless I specifically try to make it happen. It could be that I try to pick an epoch that is in as many tasks as possible then use that repeatedly. Make sure this happens in sequence is a bit tricky too. Maybe you should keep a map of epochs to mdps? It would cover more cases than just comparing to the last one.